start.sh

```diff
@@ -352,31 +352,31 @@ if [ "$cache" == "" ]; then
     print_debug "cache: $cache"
 fi
 
-if [ "$bux_server" == "" ]; then
-    ask_for_yes_or_no "Do you want to run Bux-server?"
-    bux_server="$choice"
-    print_debug "bux_server: $bux_server"
+if [ "$spv_wallet" == "" ]; then
+    ask_for_yes_or_no "Do you want to run spv-wallet?"
+    spv_wallet="$choice"
+    print_debug "spv_wallet: $spv_wallet"
 fi
 
-if [ "$pulse" == "" ]; then
-    ask_for_yes_or_no "Do you want to run Pulse?"
-    pulse="$choice"
-    print_debug "pulse: $pulse"
+if [ "$block_headers_service" == "" ]; then
+    ask_for_yes_or_no "Do you want to run Block Headers Service?"
+    block_headers_service="$choice"
+    print_debug "block_headers_service: $block_headers_service"
 fi
 
-if [ "$bux_wallet_frontend" == "" ]; then
-    ask_for_yes_or_no "Do you want to run bux-wallet-frontend?"
-    bux_wallet_frontend="$choice"
-    print_debug "bux_wallet_frontend: $bux_wallet_frontend"
+if [ "$wallet_frontend" == "" ]; then
+    ask_for_yes_or_no "Do you want to run wallet-frontend?"
```

Comment:
As this is question to user, this should mention full name.
```suggestion
    ask_for_yes_or_no "Do you want to run spv-wallet-web-frontend?"
```

Reply:
changed to full name